index.js

```diff
@@ -89,7 +89,7 @@
   if (typeof module !== 'undefined') {
     module.exports = format;
   } else if (typeof define === 'function' && define.amd) {
-    define(format);
+    define(function () { return format; });
```

Comment:
done

On Mon, Jun 27, 2016 at 4:41 PM Kevan Davis <kmdavis.lordcerberus@gmail.com>
wrote:

> ok
>
> On Mon, Jun 27, 2016 at 4:40 PM David Chambers <notifications@github.com>
> wrote:
>
>> In index.js
>> <https://github.com/davidchambers/string-format/pull/19#discussion_r68661495>
>> :
>>
>> > @@ -89,7 +89,7 @@
>> >    if (typeof module !== 'undefined') {
>> >      module.exports = format;
>> >    } else if (typeof define === 'function' && define.amd) {
>> > -    define(format);
>> > +    define(function () { return format; });
>>
>> Thanks for the update. Would you mind squashing? I don't like to have
>> failing tests at any point in a repository's history.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/davidchambers/string-format/pull/19/files/4438cd52d2311de61a6109017ab9f5e616084521#r68661495>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AAIGxnHAMaxOxzOz-4hOa8jtKSBTR3AOks5qQENggaJpZM4I_hEN>
>> .
>>
>